src/_scss/pages/state/overview/_details.scss

```diff
@@ -1,72 +1,45 @@
 .details {
     @import './detailsTooltip';
 
-    .details__table {
-        margin-top: rem(10);
-        margin-bottom: rem(15);
-
-        tbody {
-            tr {
-                &:nth-child(odd) {
-
-                    td,
-                    th {
-                        background-color: $color-gray-lightest;
-                    }
-                }
-
-                td {
-                    border: 0;
-                    font-weight: $font-light;
-                    padding: rem(8) rem(15);
-                }
-
-                th {
-                    font-weight: $font-normal;
-                    border: 0;
-                    padding: rem(8) rem(15);
-                    vertical-align: top;
-
-                    .recipient-congressional-district-tooltip {
-                        @include display(inline-block);
-                        width: 100%;
-                    }
-                }
-
-                th.details__table-cd-row {
-                    display: flex;
-                    line-height: 1.5;
-                    align-content: center;
-                    
-                    .details__table-cd-text {
-                        white-space: nowrap;
-                        @media(max-width: $medium-screen) {
-                            white-space: normal;
-                        }
-                    }
+    svg {
+        height: 14px;
+    }
 
-                    .tooltip-wrapper {
-                        @media(min-width: $tablet-screen) and (max-width: $medium-screen) {
-                            svg {
-                                margin-left: 0;
-                            }
-                        }
-                        @media(min-width: $large-screen) {
-                            padding: 0;
-                            margin: 0;
-                        }
-                        .tooltip-spacer {
-                            transform: translateX(-24px) !important;
-                            @media(min-width: 596px) {
-                            transform: translateX(-160px) !important;
-                            }
-                            @media(min-width: $tablet-screen) {
-                                transform: translate(-172px, 8px) !important;
-                            }
-                        }
-                    }
-                }
-            }
+    margin-left: 7px;
+    margin-top: 36px;
+
+    .details-info {
+        .details-header {
+            width: 351px;
+            height: 20px;
+            flex-grow: 0;
+            margin: 0 0 2px;
+            font-size: 13px;
```

Comment:
need to use rems for font-size for accessibility scan.  in our project 1 rem is 10 px.  so this should be updated to 1.3rem